examples/assets/icons.ts

```diff
@@ -0,0 +1,7 @@
tFWdlpC51r3gD7Zll1DLnclajfV70PuREiV+8ZeuTWG6LvzJYWo3o9RGXOIbpRRtW6MGlXxOM3ZRTuMoshQP97NyUdi81bTIm1KeUFPFmB/ndHbTyyI2EIeIYepqlq56heOJKamOupXRtazGvX16ZziDb6Awvfjj0w3crYC2bF/a/HDsHeD4uAZ+jDIh+4XmZHOoepc+cp9R8dzt0ModBRB6cgBOiLE3D1LGyZIC536+ZG9MXzus3sfCYgAikNyG7RCggwgKrPSMxTujDpb8QjMb+/AoSuomAC9MnXU+TpmB5ulEq6nkB0m6Re5OJkAl0R8CH3rjA5UxOBnfiekphfTDwbWMybWjjYV9run7i8IXZ2MNc7uavbQXfvlMHrTGA0Ac/QRxPx/20JMBOanZW3YrogLoX0e4LYAUF4LoVgco+B/jkLDHSh3PaJsNA1HUvRP/UsBCcTGJOAZ+hjInKGJgJ78j0lMd/WYt7UusG/0pYvEsKOmO5VTyHpmfb7pRCIY6iGgGfo1XAOXwuzH13RrltpdL4yetJh9k0t5tGbsbX/IzP101i7cescoZY+gbcT6Kv6dDKBjgQ8Q++IxyubCOjVqCmIuS6Ae7fFvKllE/tK22qm/l7sqgRCm48Y9k8gDodQAQHP0CuAHL0KZjzzEMOd2ALBY7kL//W8bF/NHrwhu3Gffrsw+S7Bot/SpnPoE+i3j3YTt/PkS8Az9HzbvpfIdR4vupjrAiPfmtZLqwfPiwA+rDbHHggeytz4f0DwGOx+BQQ8Q68AcuQqmOXMgf93YwsGjkOPc92QAf6KwDHY9T4J0Id1C9iFWOTHxGqWvgR9+Mk+MXizDAh4hp5BIw8Y4q5sH1nMFf5HLOYD9oLAm9P2ekiLnp8wJXAYmqXvEdh/u14BAc/QK4ActQpmNvrBdwu2dNQY8PtoBvS9Avtv1wsiQH8+gqI+U1BxwyjmPiqdQH9+aRiVu876E/AMvf5tNEwPt6HyyGJ+Lf5/apgAXXetCByMN3+plUe9OaNnQOzQ2ybOnRMBz9Bzau0eY2VGcymbrNvjZnXJrnOOazCbubUuDtmP4ROgTy+JF/qhN//wvenLg+vo06v2taU3Sp6AZ+jJN3F/ATLwrcWWUcVcQe9hMe+v7VPeij5xN/HtEjjGVdg3Nwzsv10vkYAFvUS4wYuOfAHOjxi4jw/O3+6XRIC+oWf4H1lS8VUUG3nfrIJPtnX4kHu2Td8+cGYAuqJW9+7qca/R0k04vBqD9vPRHLe/1RGgj4+jtr9iEQ9fv4Dfi9HH/aCZ6rpMiJo8Qw/RTJU7qRdcRBRzgfqExbzy/hKuQvqIrhTXy4Yi3so2K37vHA66HS6dgAW9dMQhK9g9pNczzPBrBupzg/putysmQF/RFe/HVFxtUdXtVlRBLicdAj7knk5bFhIJhyJ1CPKaQgqrtpCnqO4tDNLRH/NZLbXMa6O/66FJN2MRH560Pv39ksyb0OE3EfAMvQmGv04lEPVQ3hct5u7BvRKgz+g89EG9bleT/FH31ZrgS88Nz9DTa9O+I2K2ov5wN7Z434UMZ0PdVzyRwfmV4VTvWiMTGOn3mulGu01TLxxahH7/cmT+9r04Ap6hF8cyhZImEUQ0MddFTXtZzFPofsOJgb4ztQ9Re7QfhAvh80bDoeZa60jAgl7HVhmeTx8YXtV91/wrBuTL+t7aG5oABOhDOsoT8QK5iPus+1xJBHzIvSSw0YrlsKN+3N2LLRLId82sVmQwvjGQz3a1pgTYBybgml5GNHNNXWzl1uMsfAP7gF/Y0opOZss8Q8+swTuEuwHrIom5QjnJYt6hRb2qJwL0pTvZ4Nc9bTT8zHom/SbDd8Me1IGABb0OrVAPH95dDzd68uJ/e8rtzCYwNgH1qVfHzlarHBH33VoBTMUZC3oqLTl4HFsMXkSlJZzBjOrqSmt0ZckToE/p0cEnBQs02r4bDG8cd30OPU5bleYp5w6XpPC7SqugnIL9UI1yuGZfKvtDxIcrrcCPkX9k33iZA/AMPfMOMBL+5sEwnM/g5SdkBWu0KO7St3TF+2lR/B3xM9o+HAxvDHct6DHaqWwvox2yO6xsIC4/ewLR+li0fTj7DlYGAB9yL4NqoDI5vDgOd/X4y7mDuH0Pfo5nFhXtwqUgeO1mgwD7hp7xvmzj/5p/voB/C7BfPFtzP+1eiQQ8Qy8RbpCiV8PPKGIupL+0mAfpWfHdPDZQCHql6tqB/LWrJRCwoJcANViR6wXz9xfB/LW7cQlI0PXwoigp2r4chWsYPy3oYZqqNEfXL63k4gu+ktm5r+QtnqtLbEGAvqY7Py5ssaquiyzodW2ZivyyoFcEusbV6IUsUZJn51FaKh0/Ix12X5fz/h7T0+l7PUfii+J6RpbOBuz8SxPNbUEi0isiF2XW9HAQf+1mAgTYR+YhjAex2YOEswr7yPVBfLWbBRPwr7mCgQYrLtIhOj0ZzmIerINFd5c+9xQxnBoojkhH3AJhjeGqBT1GO5Xl5eplFVxCub8poUwXaQLdEIjU99boJiDnSZOABT3Ndu02qpW6zViDfOfWwAe7kCeB8wk7ynMPIu3TefamEqO2oJcIN0DRKwfwUS7ewqHP+4L4ajcTI0Dfe4yQrgsS1oqc9/e1UUEaq2g3LehFEw1SHju93n2+UBB3PTsP0lAJuxmlD85JG+hiV6cMCVjQM2z0kZAjHZqLMpjm25vSjzxSH4y0b6ffcyqM0IJeIeyaVRXlcLue1HVezdjZnfwIXETIunUyQoqyb0dgGcpHC3qo5irU2eUKLa28wm7gHOYj5RXvkk1gbAL0wX+R64qxc9YiR5R9uxawUnLCgp5Sa/YWy/jesg8td6RDnUOD5IorIRDlSNH4Smi4ktoRsKDXrkkqc2hCZTUNVtFlg23urU2gMAKXFlZSuQVF2bfLpZBh6Rb0DBudK9zV7ksGCf2mIH7azfQJROmLb2Afny395nCEowlY0EcTyeP/xQhzXIBQdU
```

Comment:
these base64 encoded images look rather large.  I bet if you downsample the png to the largest size it needs to render to (60px or whatever), this would be a lot smaller.  I'm pretty sure that some browsers have a limit on the length of data urls.